tests/test_prism.py

```diff
@@ -309,16 +311,17 @@ def test_prism(schema_path, xlsx_path):
 
     # turn into object.
     ct, file_maps = prismify(xlsx_path, schema_path, assay_hint=hint)
+    ct['lead_organization_study_id'] = 'test_prism_trial_id'
```

Comment:
Maybe before modifying it, we could assert it's parsed correctly from the example file?
```suggestion
    if hint = 'cytof':
        assert "CYTOF_TEST1" == ct['lead_organization_study_id']
        ct['lead_organization_study_id'] = 'test_prism_trial_id'
```